cypress/integration/query/create_query_spec.js

```diff
@@ -1,13 +1,33 @@
 describe('Create Query', () => {
   beforeEach(() => {
+    const pg = {
+      name: 'test',
+      options: {
+        dbname: 'postgres',
+        host: 'postgres',
+        password: 'postgres',
+        user: 'postgres',
+      },
+      type: 'pg',
+    };
+
     cy.login();
+    cy.request({
```

Comment:
I've created [db-seed.js](https://github.com/getredash/redash/blob/master/cypress/seed-data.js) with this purpose :thinking:, so this kind of dependency could be created by using `npm run cypress db-seed`, lmk what you think haha

Reply:
@gabrieldutra I was in fact, just testing. Though I could use some help. I cannot reproduce this percy issue locally that shows up here. In fact, when I run the `create_query_spec.js` test on `master` locally, the DOM snapshots seem to be missing the shema data (included screenshot below) And on the other hand, the snapshot for this PR seems to show the schema locally.

Any idea what might be going on here?

<img width="1387" alt="screen shot 2019-01-21 at 5 18 28 pm" src="https://user-images.githubusercontent.com/784781/51501647-cb0b5080-1da0-11e9-84b0-0187c1b805f6.png">

Reply:
Also, I don't know if it's related, but I noticed the **Chinook** data source is not showing schema info in the [preview](https://deploy-preview-3291--redash-preview.netlify.com/queries/new).

I'll try to reproduce this locally and give you some help with Percy anyway

Reply:
> I have faced some issues when I was doing changes in frontend and handling Cypress in development mode. The frontend code seems not to be shared within the docker container.

Does the Cypress Docker Compose configuration use VOLUMEs?

Reply:
Cypress is using `docker-compose.cypress.yml` when in CI and the development `docker-compose.yml` when not.

However it uses `http://localhost:5000`, which I guess doesn't use webpack to watch files, so frontend in this case only updates after a rebuild. The two options I see to make it friendlier to the developer would be either adding a `npm run start` to a frontend container in `docker-compose.yml` or adding this outside docker in cypress scripts.

Reply:
I think we should add to the instructions to run `npm run build` before running Cypress tests. Running npm in the container is not possible, because the container will not have Node (currently it does, but it's a temporary thing).